source/plugin/project/version-numbers.rst

```diff
@@ -58,7 +56,7 @@ Timestamp
 
 This is a less commonly used versioning scheme, but it is also a very simple one, as it does not contain any additional
 information in the version numbers. It does not matter which separators you use (if any) or which date and time parts
-you include. However we recommend you to order date and time parts in the descending order of duration like
+you include. However, we recommend you to order date and time parts in the descending order of duration like
```

Comment:
Whilst we're quibbling this sentence, I'd replace "you to order" with "ordering" (or "that you").
The "the" in front of "descending" isn't needed, too.